examples/flex-flows/unify-ai/code_quality_unify_ai.py

```diff
@@ -0,0 +1,19 @@
+import pathlib
+import sys
+
+from promptflow.core import OpenAIModelConfiguration
+
+# Add the path to the evaluation code quality module.
+eval_code_quality_path = str(pathlib.Path(__file__).parent / "../eval-code-quality")
```

Comment:
This won't work in cloud since we currently will only upload current working directory to cloud, import from outside of current directory will fail with module not found error

Reply:
ok, I have copied the code from `eval-code-quality` to mitigate this issue